### PR TITLE
Minor JavaScript syntax edits

### PR DESCRIPTION
**Pull request by arkon**
Did a quick pass-through of the JavaScript code while trying to familiarize myself with the code base.

### CODE REVIEW

File: public/js/graph/sidebar/fce_count.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 function updateFCECount() {
     'use strict';
 
-    FCEs = FCEs100 + FCEs200 + FCEs300 + FCEs400 + FCEsMAT;
+    var FCEs = FCEs100 + FCEs200 + FCEs300 + FCEs400 + FCEsMAT;
```
Comment (elliottsj) [inline public/js/graph/sidebar/fce_count.js]: Careful with this one; a lot of global variables are intended to be global

File: public/js/graph/sidebar/sidebar_events.js [hunk 2]
```diff
@@ -120,14 +120,9 @@ function getGraphsInDatabase() {
...
 function enableReset() {
-    'use strict'; 
+    'use strict';
 
-    if (FCEs > 0) {
-        $('#reset').attr('disabled', false);
-    } else {
-        $('#reset').attr('disabled', true);  
-    }
+    $('#reset').attr('disabled', FCEs <= 0);
```
Comment (elliottsj) [inline public/js/graph/sidebar/sidebar_events.js]: See https://github.com/Courseography/courseography/pull/586/files#r29263284

File: public/js/graph/sidebar/fce_count.js (hunk 1 above)
Comment (arkon) [inline public/js/graph/sidebar/fce_count.js]: Touché.

**Comment by arkon** [comment]
I guess it should also be noted that there's a _lot_ of redundancy when there's functions dealing with minors/majors/specialists or 300-level/400-level sort of things.

**Comment by david-yz-liu** [comment]
Thanks, @arkon.

**Pull request merged**